### PR TITLE
[tests-only][full-ci] Add spaces example for apiTranslation suite

### PR DESCRIPTION
**Pull request by SagarGi**
## Description
 Added Spaces for suite apiTranslation 

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/3101

## How Has This Been Tested?

- CI
- Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by amrita-shrestha** [comment]
LGTM

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

File: tests/acceptance/features/apiTranslation/translation.feature [hunk 1]
```diff
@@ -19,10 +19,18 @@ Feature: translate messages in api response to preferred language
...
+      | old         | es-ES    |
       | old         | zh-CN    |
-      | new         | zh-CN    |
       | old         | fr-FR    |
-      | new         | fr-FR    |
-      | old         | es-ES    |
+      | new         | de-DE    |
       | new         | es-ES    |
+      | new         | zh-CN    |
+      | new         | fr-FR    |
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiTranslation/translation.feature:28]: hmm...reordered the example list: old after new :+1:

**Review by kiranparajuli589 — COMMENTED** [review]
looks good

File: tests/acceptance/features/apiTranslation/translation.feature (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/apiTranslation/translation.feature:28]: > hmm...reordered the example list: old after new +1

new is after old - that is good.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**